apps/central-scan/backend/src/app.diagnostics.test.ts

```diff
@@ -23,20 +28,36 @@ jest.mock(
   })
 );
 
+jest.mock(
+  '@votingworks/printing',
+  (): typeof import('@votingworks/printing') => ({
+    ...jest.requireActual('@votingworks/printing'),
+    createSimpleRenderer: jest.fn(),
+  })
+);
+
```

Comment:
Is there a way to avoid mocking this, but instead toggle the `jest` timers at appropriate times in the test to get the desired behavior?